HTTP Server.core/Reply.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Text;
+
+namespace HTTPServer.core
+{
+    public static class Reply
+    {
+        public static byte[] HandleReply(byte[] request, IDirectoryContents directoryContents)
+        {
+            var requestMessage = Encoding.UTF8.GetString(request).Substring(0, request.Length);
+            var uri = "";
+            var method = "";
+            var version = "";
+            try
+            {
+                string[] requestLine = requestMessage.Split(' ', ' ');
+                method = requestLine[0];
+                uri = requestLine[1];
+                version = requestLine[2];
+            }
+            catch
+            {
+                return Encoding.UTF8.GetBytes(CreateMessage("HTTP/1.1 400 Bad Request\r\n", directoryContents));
```

Comment:
i see this phrase repeated in this method a number of times - is there a better way to do this?